<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mugi-ffmpeg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="proceed">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Enqueue</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Inputs and outputs</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,3,0,0" rowminimumheight="0,0,0,0,0">
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="addVideo">
               <property name="text">
                <string>video</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addAudio">
               <property name="text">
                <string>audio</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addSubtitles">
               <property name="text">
                <string>subtitles</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="deleteAllSources">
               <property name="text">
                <string>delete all</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Sources</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Files</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Output</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTableView" name="sources">
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QListWidget" name="files"/>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="selectAllFiles">
               <property name="text">
                <string>all</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="unselectAllFiles">
               <property name="text">
                <string>none</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="output">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="selectOutputDirectory">
               <property name="text">
                <string>select</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,0">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="2" column="1" colspan="4">
            <widget class="QLineEdit" name="optionsAudio">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>-c:a copy</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="scale720">
             <property name="text">
              <string>scale -1:720</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Other</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="vorbis">
             <property name="text">
              <string>vorbis</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="noScale">
             <property name="text">
              <string>no scale</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="4">
            <widget class="QLineEdit" name="optionsVideo">
             <property name="text">
              <string>-c:v copy</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="4">
            <widget class="QLineEdit" name="optionsOther">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Video</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Audio</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="x264">
             <property name="text">
              <string>x264</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="videoCopy">
             <property name="text">
              <string>copy</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="audioCopy">
             <property name="text">
              <string>copy</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="enqueue">
          <property name="text">
           <string>enqueue</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Run</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,1,0">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Tasks</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableView" name="tasks">
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>70</number>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="deleteCompleteTasks">
               <property name="text">
                <string>delete complete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteAllTasks">
               <property name="text">
                <string>delete all</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Progress</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QProgressBar" name="progressTotal"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPlainTextEdit" name="log"/>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="time">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="overwrite">
            <property name="text">
             <string>overwrite existing</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="run">
            <property name="text">
             <string>run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopAfterCurrent">
            <property name="text">
             <string>stop after current</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="abort">
            <property name="text">
             <string>abort</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
